### PR TITLE
Add types for fake-diff

### PR DESCRIPTION
**Pull request by mondeja**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Review by domdomegg — APPROVED** [review]
LGTM

It might be worth for the sake of developer experience to extract the options into their own interface type, which you can add comments to explain what they mean and state the defaults.

File: types/fake-diff/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Type definitions for fake-diff 1.0
+// Project: https://github.com/onurkerimov/fake-diff
+// Definitions by: Álvaro Mondéjar Rubio <https://github.com/mondeja>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = fakeDiff;
+
+interface fakeDiffOptions {
```
Comment (domdomegg) [inline types/fake-diff/index.d.ts:8]:
```suggestion
interface FakeDiffOptions {
```

nit: by convention

**Review by domdomegg — APPROVED** [review]

**Comment by mondeja** [comment]
Thanks for the review @domdomegg! 👍🏼

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@mondeja thx!

**Comment by domdomegg** [comment]
Ready to merge

**Pull request merged**